openmetadata-ui/src/main/resources/ui/playwright/e2e/Flow/ApiCollection.spec.ts

```diff
@@ -0,0 +1,122 @@
+/*
+ *  Copyright 2024 Collate.
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+import { test } from '@playwright/test';
+import { ApiCollectionClass } from '../../support/entity/ApiCollectionClass';
+import { EntityDataClass } from '../../support/entity/EntityDataClass';
+import { createNewPage, redirectToHomePage } from '../../utils/common';
+import {
+  addMultiOwner,
+  addOwner,
+  removeOwner,
+  updateOwner,
+} from '../../utils/entity';
+
+const entity = new ApiCollectionClass();
+
+// use the admin user to login
+test.use({ storageState: 'playwright/.auth/admin.json' });
+
+test.describe('API Collection Entity Special Test Cases', () => {
+  test.beforeAll('Setup pre-requests', async ({ browser }) => {
+    const { apiContext, afterAction } = await createNewPage(browser);
+
+    await EntityDataClass.preRequisitesForTests(apiContext);
```

Comment:
If you just need owner, can you directly create 2 new owner, this function creates lot of things like glossary etc

Reply:
Thanks, @ShaileshParmar11. I'm using both user and teams there, but I'll take a look and update it in my next patch.